src/WebJobs.Script/Rpc/Configuration/LanguageWorkerConfigOptionsSetup.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    internal class LanguageWorkerConfigOptionsSetup : IConfigureOptions<LanguageWorkerConfigOptions>
+    {
+        private IEnumerable<WorkerConfig> _languageWorkerConfigs;
+        private IConfiguration _configuration;
+
+        public LanguageWorkerConfigOptionsSetup(IConfiguration configuration, ILoggerFactory loggerFactory)
+        {
+            _configuration = configuration;
+            ILogger logger = loggerFactory.CreateLogger("Host.LanguageWorkerConfigOptionsSetup");
```

Comment:
I think all the actual work you're doing here should be moved from the ctor to the Configure method below.